src/main/java/com/google/opensesame/servlets/MentorBuilder.java

```diff
@@ -6,10 +6,9 @@
   private String name;
   private String gitHubID;
   private String description = "";
-  private ArrayList<String> interestTags = new ArrayList<String>;
-  private ArrayList<String> projectIDs = new ArrayList<String>;
+  private ArrayList<String> interestTags = new ArrayList<String>();
+  private ArrayList<String> projectIDs = new ArrayList<String>();
```

Comment:
Aha. I wondered earlier whether it worked without `()`, thought maybe it was some new Java feature I hadn't seen haha.

Reply:
No new feature, just my own error :) Glad to have it fixed now.